test/test_operations.py

```diff
@@ -1231,6 +1231,32 @@ def test_inplace_view_non_contig(self):
     x.sum().backward()
     self.assertEqual(root.grad.tolist(), [[1, 2], [1, 1], [1, 1]])
 
+  def test_view_data_update(self):
+    a = torch.zeros(4, device=xm.xla_device())
+    v = a.view(2, 2)
+    a.data = a.data + 1
+    self.assertEqual(a.tolist(), [1, 1, 1, 1])
+    # Upadting a.data should not update v's value.
+    self.assertEqual(v.tolist(), [[0.0, 0.0], [0.0, 0.0]])
+
+  def test_view_out_computation(self):
```

Comment:
hmm was the code in this test actually broken before? Or are you just adding this test for completeness. (this code doesn't use the `tensor.data` field, so I wouldn't expect it to be affected by your change to `XLATensor::ShallowCopyTo`)

Reply:
nope, just a test case I think is nice to have